'invoke runtests' complains about a `SECRET_KEY` stemming from opencivicdata

My system stats
```
Software:

    System Software Overview:

      System Version: OS X 10.11.6 (15G1421)
      Kernel Version: Darwin 15.6.0
      Boot Volume: Macintosh HD
      Boot Mode: Normal
      Computer Name: ivan
      User Name: Emmanuel Obi (emmanuel)
      Secure Virtual Memory: Enabled
      System Integrity Protection: Enabled
      Time since boot: 21 days 6:54
```

was running a python3.6 virtualenv as confirmed visually by @r-wei and got the following after running `$ invoke runtests`.

```
no pupa_settings on path, using defaults
2018-02-12 18:10:47 [scrapy.utils.log] INFO: Scrapy 1.4.0 started (bot: documenters_aggregator)
2018-02-12 18:10:47 [scrapy.utils.log] INFO: Overridden settings: {'BOT_NAME': 'documenters_aggregator', 'CLOSESPIDER_ERRORCOUNT': 5, 'COMMANDS_MODULE': 'documenters_aggregator.commands', 'COOKIES_ENABLED': False, 'NEWSPIDER_MODULE': 'documenters_aggregator.spiders', 'SPIDER_MODULES': ['documenters_aggregator.spiders'], 'USER_AGENT': 'Documenters Aggregator [development mode]. Learn more and say hello at https://city-bureau.gitbooks.io/documenters-event-aggregator/'}
2018-02-12 18:10:47 [invoke] DEBUG: Adding <parser/Context 'genspider': {'name': <Argument: name (n) *>, 'long-name': <Argument: long_name (l) *>, 'start-urls': <Argument: start_urls (s) *>}>
2018-02-12 18:10:47 [invoke] DEBUG: Adding <parser/Context 'runtests'>
2018-02-12 18:10:47 [invoke] DEBUG: Adding <parser/Context 'ecs.create-logs'>
2018-02-12 18:10:47 [invoke] DEBUG: Adding <parser/Context 'ecs.delete-logs'>
2018-02-12 18:10:47 [invoke] DEBUG: Adding <parser/Context 'ecs.build'>
2018-02-12 18:10:47 [invoke] DEBUG: Adding <parser/Context 'ecs.run'>
2018-02-12 18:10:47 [invoke] DEBUG: Adding <parser/Context 'ecs.login'>
2018-02-12 18:10:47 [invoke] DEBUG: Adding <parser/Context 'ecs.tag'>
2018-02-12 18:10:47 [invoke] DEBUG: Adding <parser/Context 'ecs.push'>
2018-02-12 18:10:47 [invoke] DEBUG: Adding <parser/Context 'ecs.register-task-definitions'>
2018-02-12 18:10:47 [invoke] DEBUG: Adding <parser/Context 'ecs.enable-rules'>
2018-02-12 18:10:47 [invoke] DEBUG: Adding <parser/Context 'ecs.disable-rules'>
2018-02-12 18:10:47 [invoke] DEBUG: Adding <parser/Context 'ecs.deploy'>
2018-02-12 18:10:47 [invoke] DEBUG: Parsing tasks against <Collection 'tasks': genspider, runtests>
2018-02-12 18:10:47 [invoke] DEBUG: Initialized with context: <parser/Context: {'complete': <Argument: complete [bool]>, 'debug': <Argument: debug (d) [bool]>, 'write-pyc': <Argument: write-pyc [bool]>, 'echo': <Argument: echo (e) [bool]>, 'config': <Argument: config (f)>, 'help': <Argument: help (h) ?>, 'hide': <Argument: hide>, 'list': <Argument: list (l) [bool]>, 'pty': <Argument: pty (p) [bool]>, 'version': <Argument: version (V) [bool]>, 'warn-only': <Argument: warn-only (w) [bool]>, 'collection': <Argument: collection (c)>, 'no-dedupe': <Argument: no-dedupe [bool]>, 'root': <Argument: root (r)>}>
2018-02-12 18:10:47 [invoke] DEBUG: Available contexts: {'genspider': <parser/Context 'genspider': {'name': <Argument: name (n) *>, 'long-name': <Argument: long_name (l) *>, 'start-urls': <Argument: start_urls (s) *>}>, 'runtests': <parser/Context 'runtests'>, 'ecs.create-logs': <parser/Context 'ecs.create-logs'>, 'ecs.delete-logs': <parser/Context 'ecs.delete-logs'>, 'ecs.build': <parser/Context 'ecs.build'>, 'ecs.run': <parser/Context 'ecs.run'>, 'ecs.login': <parser/Context 'ecs.login'>, 'ecs.tag': <parser/Context 'ecs.tag'>, 'ecs.push': <parser/Context 'ecs.push'>, 'ecs.register-task-definitions': <parser/Context 'ecs.register-task-definitions'>, 'ecs.enable-rules': <parser/Context 'ecs.enable-rules'>, 'ecs.disable-rules': <parser/Context 'ecs.disable-rules'>, 'ecs.deploy': <parser/Context 'ecs.deploy'>}
2018-02-12 18:10:47 [invoke] DEBUG: Wrapping up context None
2018-02-12 18:10:47 [invoke] DEBUG: Starting argv: ['runtests']
2018-02-12 18:10:47 [invoke] DEBUG: Handling token: 'runtests'
2018-02-12 18:10:47 [invoke] DEBUG: ParseMachine: 'context' => 'context'
2018-02-12 18:10:47 [invoke] DEBUG: Wrapping up context None
2018-02-12 18:10:47 [invoke] DEBUG: Moving to context 'runtests'
2018-02-12 18:10:47 [invoke] DEBUG: Context args: {}
2018-02-12 18:10:47 [invoke] DEBUG: Context flags: {}
2018-02-12 18:10:47 [invoke] DEBUG: Context inverse_flags: {}
2018-02-12 18:10:47 [invoke] DEBUG: ParseMachine: 'context' => 'end'
2018-02-12 18:10:47 [invoke] DEBUG: Wrapping up context 'runtests'
2018-02-12 18:10:47 [invoke] DEBUG: Resulting task contexts: [<parser/Context 'runtests'>]
2018-02-12 18:10:47 [invoke] DEBUG: Didn't see any /Users/emmanuel/OpenSource/city-scrapers/invoke.yaml, skipping.
2018-02-12 18:10:47 [invoke] DEBUG: Didn't see any /Users/emmanuel/OpenSource/city-scrapers/invoke.yml, skipping.
2018-02-12 18:10:47 [invoke] DEBUG: Didn't see any /Users/emmanuel/OpenSource/city-scrapers/invoke.json, skipping.
2018-02-12 18:10:47 [invoke] DEBUG: Merging config sources in order onto new empty _config...
2018-02-12 18:10:47 [invoke] DEBUG: Defaults: {'run': {'warn': False, 'hide': None, 'shell': '/bin/bash', 'pty': False, 'fallback': True, 'env': {}, 'replace_env': False, 'echo': False, 'encoding': None, 'out_stream': None, 'err_stream': None, 'in_stream': None, 'watchers': [], 'echo_stdin': None}, 'runners': {'local': <class 'invoke.runners.Local'>}, 'sudo': {'prompt': '[sudo] password: ', 'password': None, 'user': None}, 'tasks': {'dedupe': True, 'auto_dash_names': True, 'collection_name': 'tasks', 'search_root': None}}
2018-02-12 18:10:47 [invoke] DEBUG: Collection-driven: {}
2018-02-12 18:10:47 [invoke] DEBUG: System-wide config file (/etc/invoke.py): {}
2018-02-12 18:10:47 [invoke] DEBUG: Per-user config file (/Users/emmanuel/.invoke.py): {}
2018-02-12 18:10:47 [invoke] DEBUG: Per-project config file (/Users/emmanuel/OpenSource/city-scrapers/invoke.py): {}
2018-02-12 18:10:47 [invoke] DEBUG: Environment variable config: {}
2018-02-12 18:10:47 [invoke] DEBUG: Runtime config file has not been loaded yet, skipping
2018-02-12 18:10:47 [invoke] DEBUG: Overrides: {'run': {}, 'tasks': {}}
2018-02-12 18:10:47 [invoke] DEBUG: Modifications: {}
2018-02-12 18:10:47 [invoke] DEBUG: Deletions: {}
2018-02-12 18:10:47 [invoke] DEBUG: Examining top level tasks [<parser/Context 'runtests'>]
2018-02-12 18:10:47 [invoke] DEBUG: Tasks (now Calls) with kwargs: [<Call 'runtests', args: (), kwargs: {}>]
2018-02-12 18:10:47 [invoke] DEBUG: Expanding task-call <Call 'runtests', args: (), kwargs: {}>
2018-02-12 18:10:47 [invoke] DEBUG: Deduplicating tasks...
2018-02-12 18:10:47 [invoke] DEBUG: <Call 'runtests', args: (), kwargs: {}>: no duplicates found, ok
2018-02-12 18:10:47 [invoke] DEBUG: Executing <Call 'runtests', args: (), kwargs: {}>
2018-02-12 18:10:47 [invoke] DEBUG: Loading collection configuration
2018-02-12 18:10:47 [invoke] DEBUG: Merging config sources in order onto new empty _config...
2018-02-12 18:10:47 [invoke] DEBUG: Defaults: {'run': {'warn': False, 'hide': None, 'shell': '/bin/bash', 'pty': False, 'fallback': True, 'env': {}, 'replace_env': False, 'echo': False, 'encoding': None, 'out_stream': None, 'err_stream': None, 'in_stream': None, 'watchers': [], 'echo_stdin': None}, 'runners': {'local': <class 'invoke.runners.Local'>}, 'sudo': {'prompt': '[sudo] password: ', 'password': None, 'user': None}, 'tasks': {'dedupe': True, 'auto_dash_names': True, 'collection_name': 'tasks', 'search_root': None}}
2018-02-12 18:10:47 [invoke] DEBUG: Collection-driven: {}
2018-02-12 18:10:47 [invoke] DEBUG: System-wide config file (/etc/invoke.py): {}
2018-02-12 18:10:47 [invoke] DEBUG: Per-user config file (/Users/emmanuel/.invoke.py): {}
2018-02-12 18:10:47 [invoke] DEBUG: Per-project config file (/Users/emmanuel/OpenSource/city-scrapers/invoke.py): {}
2018-02-12 18:10:47 [invoke] DEBUG: Environment variable config: {}
2018-02-12 18:10:47 [invoke] DEBUG: Runtime config file has not been loaded yet, skipping
2018-02-12 18:10:47 [invoke] DEBUG: Overrides: {'run': {}, 'tasks': {}}
2018-02-12 18:10:47 [invoke] DEBUG: Modifications: {}
2018-02-12 18:10:47 [invoke] DEBUG: Deletions: {}
2018-02-12 18:10:47 [invoke] DEBUG: Running pre-merge for shell env loading...
2018-02-12 18:10:47 [invoke] DEBUG: Merging config sources in order onto new empty _config...
2018-02-12 18:10:47 [invoke] DEBUG: Defaults: {'run': {'warn': False, 'hide': None, 'shell': '/bin/bash', 'pty': False, 'fallback': True, 'env': {}, 'replace_env': False, 'echo': False, 'encoding': None, 'out_stream': None, 'err_stream': None, 'in_stream': None, 'watchers': [], 'echo_stdin': None}, 'runners': {'local': <class 'invoke.runners.Local'>}, 'sudo': {'prompt': '[sudo] password: ', 'password': None, 'user': None}, 'tasks': {'dedupe': True, 'auto_dash_names': True, 'collection_name': 'tasks', 'search_root': None}}
2018-02-12 18:10:47 [invoke] DEBUG: Collection-driven: {}
2018-02-12 18:10:47 [invoke] DEBUG: System-wide config file (/etc/invoke.py): {}
2018-02-12 18:10:47 [invoke] DEBUG: Per-user config file (/Users/emmanuel/.invoke.py): {}
2018-02-12 18:10:47 [invoke] DEBUG: Per-project config file (/Users/emmanuel/OpenSource/city-scrapers/invoke.py): {}
2018-02-12 18:10:47 [invoke] DEBUG: Environment variable config: {}
2018-02-12 18:10:47 [invoke] DEBUG: Runtime config file has not been loaded yet, skipping
2018-02-12 18:10:47 [invoke] DEBUG: Overrides: {'run': {}, 'tasks': {}}
2018-02-12 18:10:47 [invoke] DEBUG: Modifications: {}
2018-02-12 18:10:47 [invoke] DEBUG: Deletions: {}
2018-02-12 18:10:47 [invoke] DEBUG: Done with pre-merge.
2018-02-12 18:10:47 [invoke] DEBUG: Scanning for env vars according to prefix: 'INVOKE_', mapping: {'RUN_WARN': ['run', 'warn'], 'RUN_HIDE': ['run', 'hide'], 'RUN_SHELL': ['run', 'shell'], 'RUN_PTY': ['run', 'pty'], 'RUN_FALLBACK': ['run', 'fallback'], 'RUN_REPLACE_ENV': ['run', 'replace_env'], 'RUN_ECHO': ['run', 'echo'], 'RUN_ENCODING': ['run', 'encoding'], 'RUN_OUT_STREAM': ['run', 'out_stream'], 'RUN_ERR_STREAM': ['run', 'err_stream'], 'RUN_IN_STREAM': ['run', 'in_stream'], 'RUN_WATCHERS': ['run', 'watchers'], 'RUN_ECHO_STDIN': ['run', 'echo_stdin'], 'RUNNERS_LOCAL': ['runners', 'local'], 'SUDO_PROMPT': ['sudo', 'prompt'], 'SUDO_PASSWORD': ['sudo', 'password'], 'SUDO_USER': ['sudo', 'user'], 'TASKS_DEDUPE': ['tasks', 'dedupe'], 'TASKS_AUTO_DASH_NAMES': ['tasks', 'auto_dash_names'], 'TASKS_COLLECTION_NAME': ['tasks', 'collection_name'], 'TASKS_SEARCH_ROOT': ['tasks', 'search_root']}
2018-02-12 18:10:47 [invoke] DEBUG: Obtained env var config: {}
2018-02-12 18:10:47 [invoke] DEBUG: Loaded shell environment, triggering final merge
2018-02-12 18:10:47 [invoke] DEBUG: Merging config sources in order onto new empty _config...
2018-02-12 18:10:47 [invoke] DEBUG: Defaults: {'run': {'warn': False, 'hide': None, 'shell': '/bin/bash', 'pty': False, 'fallback': True, 'env': {}, 'replace_env': False, 'echo': False, 'encoding': None, 'out_stream': None, 'err_stream': None, 'in_stream': None, 'watchers': [], 'echo_stdin': None}, 'runners': {'local': <class 'invoke.runners.Local'>}, 'sudo': {'prompt': '[sudo] password: ', 'password': None, 'user': None}, 'tasks': {'dedupe': True, 'auto_dash_names': True, 'collection_name': 'tasks', 'search_root': None}}
2018-02-12 18:10:47 [invoke] DEBUG: Collection-driven: {}
2018-02-12 18:10:47 [invoke] DEBUG: System-wide config file (/etc/invoke.py): {}
2018-02-12 18:10:47 [invoke] DEBUG: Per-user config file (/Users/emmanuel/.invoke.py): {}
2018-02-12 18:10:47 [invoke] DEBUG: Per-project config file (/Users/emmanuel/OpenSource/city-scrapers/invoke.py): {}
2018-02-12 18:10:47 [invoke] DEBUG: Environment variable config: {}
2018-02-12 18:10:47 [invoke] DEBUG: Runtime config file has not been loaded yet, skipping
2018-02-12 18:10:47 [invoke] DEBUG: Overrides: {'run': {}, 'tasks': {}}
2018-02-12 18:10:47 [invoke] DEBUG: Modifications: {}
2018-02-12 18:10:47 [invoke] DEBUG: Deletions: {}
2018-02-12 18:10:47 [invoke] DEBUG: Finished loading collection & shell env configs
2018-02-12 18:10:47 [invoke] DEBUG: Didn't see any /etc/invoke.yaml, skipping.
2018-02-12 18:10:47 [invoke] DEBUG: Didn't see any /etc/invoke.yml, skipping.
2018-02-12 18:10:47 [invoke] DEBUG: Didn't see any /etc/invoke.json, skipping.
2018-02-12 18:10:47 [invoke] DEBUG: Didn't see any /Users/emmanuel/.invoke.yaml, skipping.
2018-02-12 18:10:47 [invoke] DEBUG: Didn't see any /Users/emmanuel/.invoke.yml, skipping.
2018-02-12 18:10:47 [invoke] DEBUG: Didn't see any /Users/emmanuel/.invoke.json, skipping.
2018-02-12 18:10:47 [invoke] DEBUG: Merging config sources in order onto new empty _config...
2018-02-12 18:10:47 [invoke] DEBUG: Defaults: {'run': {'warn': False, 'hide': None, 'shell': '/bin/bash', 'pty': False, 'fallback': True, 'env': {}, 'replace_env': False, 'echo': False, 'encoding': None, 'out_stream': None, 'err_stream': None, 'in_stream': None, 'watchers': [], 'echo_stdin': None}, 'runners': {'local': <class 'invoke.runners.Local'>}, 'sudo': {'prompt': '[sudo] password: ', 'password': None, 'user': None}, 'tasks': {'dedupe': True, 'auto_dash_names': True, 'collection_name': 'tasks', 'search_root': None}}
2018-02-12 18:10:47 [invoke] DEBUG: Collection-driven: {}
2018-02-12 18:10:47 [invoke] DEBUG: System-wide config file (/etc/invoke.py): {}
2018-02-12 18:10:47 [invoke] DEBUG: Per-user config file (/Users/emmanuel/.invoke.py): {}
2018-02-12 18:10:47 [invoke] DEBUG: Per-project config file has not been loaded yet, skipping
2018-02-12 18:10:47 [invoke] DEBUG: Environment variable config: {}
2018-02-12 18:10:47 [invoke] DEBUG: Runtime config file has not been loaded yet, skipping
2018-02-12 18:10:47 [invoke] DEBUG: Overrides: {}
2018-02-12 18:10:47 [invoke] DEBUG: Modifications: {}
2018-02-12 18:10:47 [invoke] DEBUG: Deletions: {}
====================================================== test session starts =======================================================
platform darwin -- Python 3.6.4, pytest-3.1.3, py-1.4.34, pluggy-0.4.0
rootdir: /Users/emmanuel/OpenSource/city-scrapers, inifile:
plugins: betamax-0.8.0
collected 0 items / 1 errors 

============================================================= ERRORS =============================================================
_______________________________________________________ ERROR collecting  ________________________________________________________
venv-city-scrapers/lib/python3.6/site-packages/_pytest/config.py:336: in _getconftestmodules
    return self._path2confmods[path]
E   KeyError: local('/Users/emmanuel/OpenSource/city-scrapers/venv-city-scrapers/lib/python3.6/site-packages/opencivicdata/tests')

During handling of the above exception, another exception occurred:
venv-city-scrapers/lib/python3.6/site-packages/_pytest/config.py:367: in _importconftest
    return self._conftestpath2mod[conftestpath]
E   KeyError: local('/Users/emmanuel/OpenSource/city-scrapers/venv-city-scrapers/lib/python3.6/site-packages/opencivicdata/tests/conftest.py')

During handling of the above exception, another exception occurred:
venv-city-scrapers/lib/python3.6/site-packages/_pytest/config.py:373: in _importconftest
    mod = conftestpath.pyimport()
venv-city-scrapers/lib/python3.6/site-packages/py/_path/local.py:662: in pyimport
    __import__(modname)
<frozen importlib._bootstrap>:971: in _find_and_load
    ???
<frozen importlib._bootstrap>:955: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:656: in _load_unlocked
    ???
<frozen importlib._bootstrap>:626: in _load_backward_compatible
    ???
venv-city-scrapers/lib/python3.6/site-packages/_pytest/assertion/rewrite.py:211: in load_module
    py.builtin.exec_(co, mod.__dict__)
venv-city-scrapers/lib/python3.6/site-packages/opencivicdata/tests/conftest.py:4: in <module>
    from opencivicdata.core.models import (
venv-city-scrapers/lib/python3.6/site-packages/opencivicdata/core/models/__init__.py:2: in <module>
    from .jurisdiction import Jurisdiction
venv-city-scrapers/lib/python3.6/site-packages/opencivicdata/core/models/jurisdiction.py:3: in <module>
    from django.contrib.postgres.fields import ArrayField
venv-city-scrapers/lib/python3.6/site-packages/django/contrib/postgres/fields/__init__.py:1: in <module>
    from .array import *  # NOQA
venv-city-scrapers/lib/python3.6/site-packages/django/contrib/postgres/fields/array.py:3: in <module>
    from django.contrib.postgres import lookups
venv-city-scrapers/lib/python3.6/site-packages/django/contrib/postgres/lookups.py:4: in <module>
    from .search import SearchVector, SearchVectorExact, SearchVectorField
venv-city-scrapers/lib/python3.6/site-packages/django/contrib/postgres/search.py:47: in <module>
    class SearchVector(SearchVectorCombinable, Func):
venv-city-scrapers/lib/python3.6/site-packages/django/contrib/postgres/search.py:50: in SearchVector
    _output_field = SearchVectorField()
venv-city-scrapers/lib/python3.6/site-packages/django/db/models/fields/__init__.py:172: in __init__
    self.db_tablespace = db_tablespace or settings.DEFAULT_INDEX_TABLESPACE
venv-city-scrapers/lib/python3.6/site-packages/django/conf/__init__.py:56: in __getattr__
    self._setup(name)
venv-city-scrapers/lib/python3.6/site-packages/django/conf/__init__.py:41: in _setup
    self._wrapped = Settings(settings_module)
venv-city-scrapers/lib/python3.6/site-packages/django/conf/__init__.py:129: in __init__
    raise ImproperlyConfigured("The SECRET_KEY setting must not be empty.")
E   django.core.exceptions.ImproperlyConfigured: The SECRET_KEY setting must not be empty.

During handling of the above exception, another exception occurred:
venv-city-scrapers/lib/python3.6/site-packages/py/_path/common.py:372: in visit
    for x in Visitor(fil, rec, ignore, bf, sort).gen(self):
venv-city-scrapers/lib/python3.6/site-packages/py/_path/common.py:421: in gen
    for p in self.gen(subdir):
venv-city-scrapers/lib/python3.6/site-packages/py/_path/common.py:421: in gen
    for p in self.gen(subdir):
venv-city-scrapers/lib/python3.6/site-packages/py/_path/common.py:421: in gen
    for p in self.gen(subdir):
venv-city-scrapers/lib/python3.6/site-packages/py/_path/common.py:421: in gen
    for p in self.gen(subdir):
venv-city-scrapers/lib/python3.6/site-packages/py/_path/common.py:421: in gen
    for p in self.gen(subdir):
venv-city-scrapers/lib/python3.6/site-packages/py/_path/common.py:410: in gen
    dirs = self.optsort([p for p in entries
venv-city-scrapers/lib/python3.6/site-packages/py/_path/common.py:411: in <listcomp>
    if p.check(dir=1) and (rec is None or rec(p))])
venv-city-scrapers/lib/python3.6/site-packages/_pytest/main.py:686: in _recurse
    ihook = self.gethookproxy(path)
venv-city-scrapers/lib/python3.6/site-packages/_pytest/main.py:590: in gethookproxy
    my_conftestmodules = pm._getconftestmodules(fspath)
venv-city-scrapers/lib/python3.6/site-packages/_pytest/config.py:350: in _getconftestmodules
    mod = self._importconftest(conftestpath)
venv-city-scrapers/lib/python3.6/site-packages/_pytest/config.py:375: in _importconftest
    raise ConftestImportFailure(conftestpath, sys.exc_info())
E   _pytest.config.ConftestImportFailure: ImproperlyConfigured('The SECRET_KEY setting must not be empty.',)
E     File "<frozen importlib._bootstrap>", line 971, in _find_and_load
E     File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
E     File "<frozen importlib._bootstrap>", line 656, in _load_unlocked
E     File "<frozen importlib._bootstrap>", line 626, in _load_backward_compatible
E     File "/Users/emmanuel/OpenSource/city-scrapers/venv-city-scrapers/lib/python3.6/site-packages/_pytest/assertion/rewrite.py", line 211, in load_module
E       py.builtin.exec_(co, mod.__dict__)
E     File "/Users/emmanuel/OpenSource/city-scrapers/venv-city-scrapers/lib/python3.6/site-packages/opencivicdata/tests/conftest.py", line 4, in <module>
E       from opencivicdata.core.models import (
E     File "/Users/emmanuel/OpenSource/city-scrapers/venv-city-scrapers/lib/python3.6/site-packages/opencivicdata/core/models/__init__.py", line 2, in <module>
E       from .jurisdiction import Jurisdiction
E     File "/Users/emmanuel/OpenSource/city-scrapers/venv-city-scrapers/lib/python3.6/site-packages/opencivicdata/core/models/jurisdiction.py", line 3, in <module>
E       from django.contrib.postgres.fields import ArrayField
E     File "/Users/emmanuel/OpenSource/city-scrapers/venv-city-scrapers/lib/python3.6/site-packages/django/contrib/postgres/fields/__init__.py", line 1, in <module>
E       from .array import *  # NOQA
E     File "/Users/emmanuel/OpenSource/city-scrapers/venv-city-scrapers/lib/python3.6/site-packages/django/contrib/postgres/fields/array.py", line 3, in <module>
E       from django.contrib.postgres import lookups
E     File "/Users/emmanuel/OpenSource/city-scrapers/venv-city-scrapers/lib/python3.6/site-packages/django/contrib/postgres/lookups.py", line 4, in <module>
E       from .search import SearchVector, SearchVectorExact, SearchVectorField
E     File "/Users/emmanuel/OpenSource/city-scrapers/venv-city-scrapers/lib/python3.6/site-packages/django/contrib/postgres/search.py", line 47, in <module>
E       class SearchVector(SearchVectorCombinable, Func):
E     File "/Users/emmanuel/OpenSource/city-scrapers/venv-city-scrapers/lib/python3.6/site-packages/django/contrib/postgres/search.py", line 50, in SearchVector
E       _output_field = SearchVectorField()
E     File "/Users/emmanuel/OpenSource/city-scrapers/venv-city-scrapers/lib/python3.6/site-packages/django/db/models/fields/__init__.py", line 172, in __init__
E       self.db_tablespace = db_tablespace or settings.DEFAULT_INDEX_TABLESPACE
E     File "/Users/emmanuel/OpenSource/city-scrapers/venv-city-scrapers/lib/python3.6/site-packages/django/conf/__init__.py", line 56, in __getattr__
E       self._setup(name)
E     File "/Users/emmanuel/OpenSource/city-scrapers/venv-city-scrapers/lib/python3.6/site-packages/django/conf/__init__.py", line 41, in _setup
E       self._wrapped = Settings(settings_module)
E     File "/Users/emmanuel/OpenSource/city-scrapers/venv-city-scrapers/lib/python3.6/site-packages/django/conf/__init__.py", line 129, in __init__
E       raise ImproperlyConfigured("The SECRET_KEY setting must not be empty.")
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
==================================================== 1 error in 4.31 seconds =====================================================
2018-02-12 18:10:52 [invoke] DEBUG: Received a possibly-skippable exception: <UnexpectedExit: cmd='pytest -s' exited=2>
```

--- a/tasks.py
+++ b/tasks.py
@@ -63,7 +63,7 @@ def runtests(ctx):
     """
     Runs pytest and flake8.
     """
-    run('pytest -s', pty=pty_available)
+    run('pytest -s tests', pty=pty_available)
     run('flake8 --ignore E265,E266,E501 --exclude src', pty=pty_available)
 
 

